src/Api/Endpoints/Login.php

```diff
@@ -77,56 +75,49 @@ protected function supportedVersions() : array
      */
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
-        $requestdata = json_decode((string) $request->getBody(), true);
-        if (!isset($requestdata['username']) || !isset($requestdata['password'])) {
-            return (new \LORIS\Http\Response())
-                ->withBody(
-                    new \LORIS\Http\StringStream(
-                        json_encode(
-                            array('error' => 'Missing username or password')
-                        )
-                    )
-                )
-                ->withStatus(400);
+        if (count($request->getAttribute('pathparts')) !== 1) {
```

Comment:
just out of curiosity, what is this checking exactly?

Reply:
@davidblader The idea is to check if someone entered `/login/xyz/...`. In that case, it is safe to say that it should be a NotFound response because there is nothing under `/login`
The operator could be `> 1` instead.